src/contrib/mlir/core/ngraph_dialect/ops.cpp

```diff
@@ -303,56 +303,42 @@ mlir::LogicalResult verifyOp(NGConvolutionOp* op)
                                            stridesVal[i]);
         if (resultShape[2 + i] != resDim)
         {
-            return op->emitOpError("Invalid result spatial shape");
+            return op.emitOpError("Invalid result spatial shape");
         }
     }
     return mlir::success();
 }
 
 template <>
-mlir::LogicalResult verifyOp(NGMatMulOp* op)
+mlir::LogicalResult verifyOp(NGSoftMaxOp op)
```

Comment:
Why the renames ?

Reply:
That's probably git. I removed those that became or were unused. I thing we should remove all of these functions in a separate commit. They are growing, not doing anything and we have to maintain them.